app/src/main/res/layout/boot_wallet_error_fragment.xml

```diff
@@ -5,7 +5,7 @@
     <data xmlns:android="http://schemas.android.com/apk/res/android">
         <variable
             name="model"
-            type="org.kinecosystem.kinit.viewmodel.CreateWalletViewModel"/>
+            type="org.kinecosystem.kinit.view.bootWallet.BootWalletActions"/>
```

Comment:
BootWalletActions is the model ?? @katzio

Reply:
suggest to rename to BootWalletViewModel

Reply:
that's no a model, that's an interface of 2 models - CreateWalletViewModel & MigrateWalletViewModel